docs/pages/includes/tpm-joining-background.mdx

```diff
@@ -0,0 +1,31 @@
+The `tpm` join method is a secure way for Bots and Agents to authenticate with
+a Teleport cluster without using any shared secrets as with the`token` join
+method. Instead of using a shared secret, the unique identity of the host's
+Trusted Platform Module (TPM) and public key cryptography is used to
+authenticate the host.
+
+In environments where there is no other form of identity available to machines,
+e.g on-prem, this is the most secure method for joining.
+
+A Trusted Platform Module (TPM) is a secure, physical, cryptoprocessor that is
+installed on a host. TPMs can store cryptographic material and perform a number
+of cryptographic operations, without exposing the cryptographic material to the
+operating system. Each TPM has a unique key pair burned-in known as the
+Endorsement Key (EK).
+
+Some TPMs also contain an X.509 certificate for this key pair that is signed by
+the manufacturer's CA, this is known as the EK Certificate (EKCert). This
+certificate can be used by the TPM to prove to a third-party (who trusts the
+manufacturer's CA) that the TPM is genuine and abides by the TPM specification.
+
+When using the `tpm` join method, you must first "enroll" the TPM by querying
```

Comment:
Does "enroll" mean "enroll the TPM in the Teleport cluster"? If "enroll" isn't quite right, maybe it would make sense to say something like, "register the TPM with the Teleport Auth Service"?

Reply:
> maybe it would make sense to say something like, "register the TPM with the Teleport Auth Service"

Sounds good - I'll try something like that.

Reply:
258e78d2b9